src/schema/users.js

```diff
@@ -0,0 +1,16 @@
+import UserType from "./user"
+import { GraphQLList, GraphQLString } from "graphql"
+
+const Users = {
+  type: new GraphQLList(UserType),
+  description: "A list of Users",
+  args: {
+    ids: {
+      type: new GraphQLList(GraphQLString),
+    },
+  },
+  resolve: (root, options, request, { rootValue: { usersLoader } }) =>
+    usersLoader(options),
```

Comment:
In the context of an authenticated request, this would resolve with an object containing `body` and `headers`.

So I _think_ (the way you have it setup with `headers: true`), is you'd want `usersLoader(options).then(({ body }) => body)`

Reply:
Ah ok I don't actually need the headers and have removed them.  I wonder how I can test this locally, is there a tactic to pass my x-access-token to the browser?

Reply:
@eessex not sure if you have [graphiQL](https://github.com/graphql/graphiql) but it allows you to set the `x-access-token` for testing locally.

Reply:
Yea exactly using Chrome to point to GraphQL may not be sufficient b/c I'm not sure how you can pass in headers.

definitely recommend using GraphiQL! It lets you easily pass in your header (which you can access by logging in to staging.artsy.net, and in the JS console looking at `sd.CURRENT_USER.accessToken`).

Reply:
Yeah that's probably the source of your ` "Not authorized"` error. Let me know if you're still having trouble with that. Happy to walk you through setting it up.

Reply:
Yeah I am using graphiql and do have the access token, but I don't know where to set this data -- don't see a place on the UI, do I need a browser plugin or something?

Reply:
In the right hand corner there is an edit HTTP Headers button.

<img width="342" alt="screen shot 2018-03-16 at 12 47 34 pm" src="https://user-images.githubusercontent.com/5201004/37533465-47c0743a-2918-11e8-9805-87d3f26e9fab.png">

And then you can edit the value.

![drawing](https://user-images.githubusercontent.com/5201004/37533716-056bdb64-2919-11e8-9c07-25e64bf7c861.png)